skribenten-web/frontend/package.json

```diff
@@ -53,7 +53,7 @@
     "@types/react-dom": "18.2.22",
     "@typescript-eslint/eslint-plugin": "8.19.0",
     "@vitejs/plugin-react": "4.3.4",
-    "cypress": "13.17.0",
+    "cypress": "13.14.2",
```

Comment:
13.17.0 får alle cypress testene til å feile. https://github.com/cypress-io/cypress/issues/25913#issuecomment-2587390726 - denne versjonen ser ut til å fungere fint

Reply:
Okei- var sikkert noe hiccup som lot meg bruke 1 time for ingenting. 13.17.0 ser ut til å fungere fint nå